truss/tests/samples.py

```diff
@@ -189,14 +189,14 @@ def predict(self, inputs):
 
 create_reference_embeddings()
 
-mk_truss = scaffold_custom(
+create = scaffold_custom(
```

Comment:
everything about here seems to be correct. The stuff below seems to have very odd naming